src/audio/AudioResource.ts

```diff
@@ -16,6 +16,7 @@ interface CreateAudioResourceOptions<T> {
 	/**
 	 * Optional metadata that can be attached to the resource (e.g. track title, random ID).
 	 * This is useful for identification purposes when the resource is passed around in events.
+	 * @see AudioResource#metadata
```

Comment:
See is not a keyword used with typedoc for this kind of thing.

https://typedoc.org/guides/doccomments/#symbol-references